man/repart.d.xml

```diff
@@ -422,6 +422,22 @@
         <xi:include href="version-info.xml" xpointer="v247"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>MkfsOptions=</varname></term>
+
+        <listitem><para>Takes a list of command line options to be passed to <literal>mkfs</literal> at file
+        system creation time. Options are specific to the file system type specified by
+        <varname>Format=</varname>.</para>
+
+        <para>Additionally, default <literal>mkfs</literal> options can be globally set for each file system
+        format supported by <varname>Format=</varname> by setting the corresponding environement variable:
+        <literal>SYSTEMD_REPART_MKFS_OPTIONS_BTRFS</literal>, <literal>SYSTEMD_REPART_MKFS_OPTIONS_XFS</literal>,
```

Comment:
use `<varname></varname>` for env var names.

Also, please prefix the variable names with `$`. We do that pretty systematically in our docs

Reply:
Doc moved to different PR